server/service/osquery_utils/queries.go

```diff
@@ -866,5 +867,19 @@ func GetDetailQueries(ac *fleet.AppConfig, fleetConfig config.FleetConfig) map[s
 		generatedMap["scheduled_query_stats"] = scheduledQueryStats
 	}
 
+	for _, env := range os.Environ() {
+		prefix := "FLEET_DANGEROUS_REPLACE_"
+		if !strings.HasPrefix(env, prefix) {
+			continue
+		}
+		if i := strings.Index(env, "="); i >= 0 {
+			queryName := strings.TrimPrefix(env[:i], prefix)
```

Comment:
Convert to lowercase (people usually assume uppercase for env variables, and query names are lowercase)